src/test/java/com/google/api/generator/gapic/composer/grpc/goldens/GrpcTestingStub.golden

```diff
@@ -142,6 +144,11 @@ public class GrpcTestingStub extends TestingStub {
   private final GrpcOperationsStub operationsStub;
   private final GrpcStubCallableFactory callableFactory;
 
+  private static final PathTemplate PATH_TEMPLATE_GETTEST_0 =
```

Comment:
WDYT about making `GETTEST` suffix snake case (`GET_TEST`) (generator has lots of utility 
 methods to do camel-snake case conversion) and instead of adding indexes add field name in templates for easier matching of stuff and making it a bit  more readable. Also maybe make `PATH_TEMPLATE `a suffix instead of prefix.

i.e. `PATH_TEMPLATE_GETTEST_0` to become `PATH_TEMPLATE_GET_TEST_RENAME`

Reply:
Yes I have a TODO to make it proper snake style. However, I added the index on purpose because field name alone or even the combination of field name and header key is not enough to make it unique, there could be multiple patterns configured for the same field name/header key.

Reply:
I think this is resolved? https://github.com/googleapis/gapic-generator-java/pull/887#discussion_r787976169

Reply:
Yes, updated to proper snake style name.